nuitduhack/Matriochka - Step 2/README.md

```diff
@@ -0,0 +1,30 @@
+# Matriochka - Step 2
+The challenge starts with `stage2.bin` binary file found at the end of the Step 1.
+
+## Investigation
+We opened the file in IDA pro to find this screen. This time the assembly was looking very complicated, so we decided to use the mighty
+F5 button to generate a pseudocode. After looking through all the available functions. We stumbled upon this interesting function:
+
+![IDA Pro screen](idascreen.png)
+
+This is definitely the password verification function! So we started from the top to find the value of each character.
+
+For the first character, we have the line: **(_BYTE **)(a2 + 8) != 80
```

Comment:
I think this should be put inside backticks ` in order to prevent it from putting everything below it bold (same goes for the other code lines)